src/main/scala/io/iohk/ethereum/mining/BlockGenerator.scala

```diff
@@ -40,7 +40,8 @@ class BlockGenerator(blockchainStorages: BlockchainStorages, blockchainConfig: B
           ommersHash = ByteString(kec256(ommers.toBytes: Array[Byte])),
           beneficiary = beneficiary.bytes,
           stateRoot = ByteString.empty,
-          transactionsRoot = buildMpt(transactions, SignedTransaction.byteArraySerializable),
+          //not here
```

Comment:
it means that we cannot calculate here transaction root because we do not know which transactions will be valid, will update